tests/utilities/niscope_measurement/NIScopeMeasurement.serviceconfig

```diff
@@ -0,0 +1,19 @@
+{
+  "services": [
+    {
+      "displayName": "NI-Scope Measurement (Py)",
```

Comment:
The correct capitalization is "NI-SCOPE": https://www.ni.com/en/support/downloads/drivers/download.ni-scope.html#494519

This applies to English text like display names and descriptions. It doesn't apply to programmatic ids like service classes; keep using PascalCase for those.

Reply:
Sure. Thank you, I'll keep these distinctions in mind.